downloadmanager/src/main/java/cm/aptoide/pt/downloadmanager/AppDownloadManager.java

```diff
@@ -81,22 +69,46 @@ public void stop() {
     }
   }
 
+  private Observable<FileDownloadCallback> startFileDownload(DownloadAppFile downloadAppFile) {
```

Comment:
Naming. Maybe handleStartFileDownload? because the startFileDownload is then called later on the chain (other class, but still)

Reply:
The problem is that we still have the handleFileDownload progress.. this method will manage the download of the file in the app download manager side, while the other one that is being called will manage the file download on the fileManager side.